create_live_iso

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+bas64_str=`base64 -w 0 handle-user-data`
```

Comment:
What determines the execution order of the init scripts if we have more than one?